src/Microsoft.Build.Tasks.Git/GitDataReader/GitRepository.cs

```diff
@@ -491,6 +491,8 @@ private static bool IsGitDirectory(string directory, out string commonDirectory)
                 try
                 {
                     commonDirectory = Path.Combine(directory, File.ReadAllText(commonLinkPath).TrimEnd(CharUtils.AsciiWhitespace));
+                    // noralize relative paths
```

Comment:
Typo in comment. I don't think the comment is needed since it's obvious what `GetFullPath` does. Also, you can simplify by calling `Path.GetFullPath()` directly on the previous line `Path.GetFullPath(Path.Combine(...))`